src/main/java/org/sagebionetworks/bridge/reporter/helper/BridgeHelper.java

```diff
@@ -51,11 +57,25 @@ public final void setBridgeCredentials(SignIn bridgeCredentials) {
 
     /*
      * Helper method to get all uploads for specified study and date range
+     * Paginated results should be added in one list altogether
      */
     public List<Upload> getUploadsForStudy(String studyId, DateTime startDateTime, DateTime endDateTime)
-            throws IOException {
-        return sessionHelper(() -> bridgeClientManager.getClient(ForWorkersApi.class).getUploadsInStudy(studyId,
-                startDateTime, endDateTime).execute().body().getItems());
+            throws IOException, InterruptedException {
```

Comment:
Rethrowing the InterruptedException will cause obscure errors to bubble up all the way back up into the PollSqsWorker. It would be better to catch the InterruptedException in the loop and log an error message instead.